scripts/run_backend_cypress.sh

```diff
@@ -1,7 +1,6 @@
-#!/bin/env bash
-script_dir=`dirname $0`
+#!/bin/bash
 
-jar="$(readlink -f $script_dir/../executable/target/executable*.jar)"
-config="$(readlink -f $script_dir/../frontend/cypress/support/backend_config.json)"
+jar="../executable/target/executable*.jar"
+config="../frontend/cypress/support/backend_config.json"
 
 java -jar $jar standalone $config
```

Comment:
Okay, it still won't work for me :-/ 

```
The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com for information on installing Java.
```

But I ran `brew install openjdk` and now this is what it looks like on my side:

<img width="583" alt="Bildschirmfoto 2022-03-09 um 17 31 56" src="https://user-images.githubusercontent.com/1403093/157486352-70831a8d-a286-4b08-b759-6d1db88d1ef0.png">

@thoniTUB Which makes me wonder: how about actually using docker to build & run the local backend / frontend?

Reply:
We cannot run docker on our Virtual Desktops 😢

Reply:
I first thought the problem is, that java is just an alias. Since they are not available by default inside a script.

Reply:
Can you check that you installed openjdk-jdk instead of openjdk-jre?

Reply:
And confirm that `JAVA_HOME` is set correctly